### DISCUSSION TITLE
two kong in k8s, the second kong failed with no configuration available

### DISCUSSION THREAD

**Discussion by yuchao-java** [Kong/kong#13675] [Help]
I deployed kong3.4 + Kong-ingress-Controller2.12 in k8s from the file:https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/v2.12.0/deploy/single/all-in-one-dbless.yaml, and it worked. 
For the needs of two kongs, I removed the sections of Service and Deployment from the file, then I changed the resource name from kong-1 to kong-2, and it failed to start with no configuration available.
The status is as follows:
NAME                                       READY   STATUS
kong-1-5b8ccd7dcb-thvx6                		1/1     Running
kong-1-controller-7c87cbb498-z47wl     		1/1     Running
kong-2-5f79c64ff7-jnppb              	    0/1     Running
kong-2-controller-6788658bbb-9gglg   		1/1     Running

When I curl pod ip:8100/status/ready
kong-1 pod returns: {"message":"ready"}
kong-2 pod returns: {"message":"no configuration available (empty configuration present)"}

It confused me, can anyone give me some help？

**Comment by mdolah-bc** [comment]
```
kong-2 pod returns: {"message":"no configuration available (empty configuration present)"}
```

Did you ever find a solution to this? I think this might have something to do with a kong deployment already running but potentially stepping on the toes of the other one. Not entirely sure though

**Reply by yuchao-java** [reply]
> ```
> kong-2 pod returns: {"message":"no configuration available (empty configuration present)"}
> ```
> 
> Did you ever find a solution to this? I think this might have something to do with a kong deployment already running but potentially stepping on the toes of the other one. Not entirely sure though

I installed two kongs with kong helm charts, which met my needs:
https://github.com/Kong/charts/tree/main/charts
